README.md

```diff
@@ -212,6 +212,38 @@ Run mev-boost pointed at our [Sepolia Relay](https://builder-relay-sepolia.flash
 `test-cli` is a utility to execute all proposer requests against mev-boost+relay. See also the [test-cli readme](cmd/test-cli/README.md).
 
 
+# Metrics
+
+Optionally the `-prometheus=true` flag can be passed to the binary to collect various prometheus metrics
+
+## MEV Metrics
+Metrics relating to mev specifics
+
+| **metric**                     | **type** | **labels** | **description**                             |
+|--------------------------------|----------|------------|---------------------------------------------|
+| mev_boost_validator_info       | gauge    | pubkey     | validator identity information              |
```

Comment:
wouldn't having a number of validators successfully registered per relay be more helpful?  per pubkey might be a lot of data for validators operating at scale